staynomad/Nomad-Front
# Added precommit hook to run Prettier

# Relevant issue

Closes #472

# Summary of change


 - `npm run format` command runs prettier
 - Husky precommit hook automatically formats code

NOTE: this PR includes prettier formatting, thus there are quite a few lines changed

# Testing/Verification

 - Frontend should run without errors
 - `npm run format` should format all code in _src_ using prettier recommended standards
 - All commits should run `npm run format` and `git add .` prior to committing

NOTE: if there are dependency tree issues with eslint version, do the following
1. Delete `package-lock.json`
2. Delete `/node_modules`
3. Remove eslint dependency from `package.json`
4. Re-install dependencies with `yarn` (not `npm`)
